.github/workflows/ruby.yml

```diff
@@ -148,32 +148,33 @@ jobs:
       matrix:
         ruby: [3.1, 3.0, 2.7, 2.5, jruby-9.2, jruby-9.3]
         rails: [7, 6_1, 6, 5_2]
-        exclude: [
-          {ruby: 3.1,       rails: 6  },
-          {ruby: 3.1,       rails: 5_2},
-          {ruby: 3.0,       rails: 6  },
-          {ruby: 3.0,       rails: 5_2},
-          {ruby: 2.7,       rails: 5_2},
-          {ruby: 2.5,       rails: 7 },
-          {ruby: jruby-9.2, rails: 7},
-          {ruby: jruby-9.3, rails: 7},
-        ]
-    services:
-      postgres:
-        image: postgres:11.6-alpine
-        env:
-          POSTGRES_DB: arelx_test
-          POSTGRES_PASSWORD: secret
-        ports:
-          - 5432:5432
-        # needed because the postgres container does not provide a healthcheck
-        options: >-
-          --health-cmd "pg_isready -d arelx_test -U postgres -p 5432"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
+        exclude:
+          [
+            { ruby: 3.1, rails: 6 },
+            { ruby: 3.1, rails: 5_2 },
+            { ruby: 3.0, rails: 6 },
+            { ruby: 3.0, rails: 5_2 },
+            { ruby: 2.7, rails: 5_2 },
+            { ruby: 2.5, rails: 7 },
+            { ruby: jruby-9.2, rails: 7 },
+            { ruby: jruby-9.3, rails: 7 },
+          ]
     steps:
       - uses: actions/checkout@v2
+      - name: setup locale
```

Comment:
Please add comments here that explains why we don't use the canned pg.  The commit message is not enough here.